src/mixins/tipUrls.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+import { BACKEND_URL } from '../popup/utils/constants';
+import { getTwitterAccountUrl } from '../popup/utils/helper';
+
+export default {
+  data: () => ({
+    verifiedUrls: [],
+    blacklistedUrls: [],
+  }),
+  async created() {
+    try {
+      this.verifiedUrls = (await axios.get(`${BACKEND_URL}/verified`)).data;
+      this.blacklistedUrls = (await axios.get(`${BACKEND_URL}/static/wallet/graylist`)).data;
+    } catch (e) {
+      this.$logError({ e, action: 'fetch-verified' });
+    }
+  },
+  methods: {
+    getStatus(tipUrl) {
```

Comment:
Can we move this method to helpers with variables defined outside of method for storing fetched lists? Like:
```js
let verifiedUrls;
let blacklistedUrls;

export async function getStatus(tipUrl) {
  if (!verifiedUrls || !blacklistedUrls) {
    const [{ data: verified }, { data: graylist }] = await Promise.all([
      axios.get(`${BACKEND_URL}/verified`),
      axios.get(`${BACKEND_URL}/static/wallet/graylist`),
    ]);
    verifiedUrls = verified;
    blacklistedUrls = graylist;
  }
  // ...
}
```
To use helper function seems clearer that to use mixin.

Reply:
Actually, I'm not sure that it is better to move it to helpers because to use async getStatus is trickier than the current sync version.

Reply:
Can we make a vuex plugin to handle this? Like we did in [Base aepp](https://github.com/aeternity/aepp-base/blob/1acbb5c448111dacddedc2efe035b754a78ce82f/src/store/plugins/ui/appsMetadata.js). So, to have a getter in vuex that will return some default value immediately and then replace it (because of getter reactivity) with actual value after fetching of corresponding data.